src/components/popups/PopupOptionsSidebar.js

```diff
@@ -5,10 +5,14 @@ import onClickOutside from 'react-onclickoutside';
 import styled from 'styled-components';
 
```

Comment:
Why is this stuff considered new changes?

Reply:
I have no idea

Reply:
lol git